test_exercise7.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from exercise7 import calculate_room_measurement_feet, convert_feet_to_meters
+
+class RoomCalculationsTest(unittest.TestCase):
+    """Calculations to get area of room"""
+
+    def test_room_measure_feet(self):
```

Comment:
Great tests, but I'd recommend adding line breaks to separate the different test phases. See these references:

https://thoughtbot.com/upcase/videos/four-phases-of-testing

https://robots.thoughtbot.com/four-phase-test